docs/src/main/asciidoc/getting-started-testing.adoc

```diff
@@ -1475,6 +1475,86 @@ You can use this tag to isolate the `@QuarkusTest` test in a specific execution
 </plugin>
 ----
 
+[NOTE]
+====
+Currently `@QuarkusTest` and `@QuarkusIntegrationTest` should not be run in the same test run.  
+
+For Maven, this means that the former should be run by the surefire plugin while the latter should be run by the failsafe plugin.
+
+For Gradle, this means the two types of tests should belong to different source sets.
+====
+.gradle custom sourceset eg.
+[%collapsible]
+====
```

Comment:
Shouldn't the snipper be inside the note?

Reply:
```
[NOTE]
====
note
[%collapsible]
====
inside
====
====
```
this was not rendering properly on https://asciidoc.org/ thats why i put it outside.

Reply:
Something like https://stackoverflow.com/a/50823475/2504224 should work

Reply:
thanks. updated.  
i haven't cloned this repo locally. can i see the rendered  version? how does the ci work?

Reply:
See: https://quarkus-pr-main-43873-preview.surge.sh/version/main/guides/getting-started-testing#mixing-quarkustest-with-other-type-of-tests